docs/developer/architecture/vsphere.md

```diff
@@ -1,3 +1,180 @@
 # vSphere
 
------
+## High-Level information
+
+### Product overview
+
+vSphere is a VMware product dedicated to managing a (usually) on-premise infrastructure. From physical machines running [VMware ESXi](https://en.wikipedia.org/wiki/VMware_ESXi) that are called ESXi Hosts, users can spin up or migrate Virtual Machines from one host to another.
+
+vSphere is an integrated solution and provides an easy managing interface over concepts like data storage, or computing resource.
+
+### Terminology
+
+This section details some of vSphere specific elements. This section does not intend to be an extensive list, but rather a place for those unfamiliar with the product to have the basics required to understand how the Datadog integration works.
+
+-   vSphere - The complete suite of tools and technologies detailed in this article.
+-   vCenter server - The main machine which controls ESXi hosts and provides both a web UI and an API to control the vSphere environment.
+-   vCSA (vCenter Server Appliance) - A specific kind of vCenter where the software runs in a dedicated Linux machine (more recent). By opposition, the legacy vCenter is typically installed on an existing Windows machine.
+-   ESXi host - The physical machine controlled by vCenter where the ESXi (bare-metal) virtualizer is installed. The host boots a minimal OS that can run Virtual Machines.
+-   VM - What anyone using vSphere really needs in the end, instances that can run applications and code. Note: Datadog monitors both ESXi hosts and VMs and it calls them both "host" (they are in the host map).
+-   Attributes/tags - It is possible to add attributes and tags to any vSphere resource, note that those two are now very similar with "attributes" being the deprecated thing to use.
```

Comment:
Is still deprecated? I thought they changed their mind

Reply:
They have feature parity now and VMware recommended migrating to tags. But AFAIK, attributes are still supported for the foreseeable future.